JSTests/wasm/stress/simple-inline-stacktrace-with-catch.js

```diff
@@ -1,3 +1,4 @@
+//@ skip if $architecture = 'arm'
```

Comment:
It should be `==`

Reply:
oh my goodness :see_no_evil:

 fixed, thanks